packages/core/content-platform/base.jsonc

```diff
@@ -0,0 +1,349 @@
+{
+  // The URL here should match the actual URL from Schema Registry
+  "$id": "https://api.myvtex.com/api/content-platform/schemas/faststore-base",
+  // This should match the URL for our consumer meta schema.
+  "$schema": "https://api.myvtex.com/api/content-platform/schemas/consumer",
+  "title": "FastStore Base Schemas",
+  "description": "Collection of base schemas for FastStore",
+  "contexts": {},
+  "components": {
+    "base-component": {
+      "type": "object",
+      "$abstract": true,
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "data": {
+          "type": "object"
+        }
+      },
+      "required": ["name"]
+    },
+    "base-section": {
+      "type": "object",
+      "$abstract": true,
+      "$extends": ["#/components/base-component"],
+      "properties": {
+        "sections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/components/base-component"
+          }
+        }
+      }
+    },
+    "base-page-template": {
+      "type": "object",
+      "$abstract": true,
+      "$extends": ["#/components/base-dynamic-area"],
```

Comment:
I think you renamed the base-dynamic-area to "base-section". Can you confirm it's correct?